serverless/aws/cdk/src/main/scala/sttp/tapir/serverless/aws/cdk/core/AppTemplateFiles.scala

```diff
@@ -0,0 +1,75 @@
+package sttp.tapir.serverless.aws.cdk.core
```

Comment:
[naming] I think in tapir mostly internal stuff is in an `internal` package

Reply:
applied 👍🏼